home work 25/front/main.js

```diff
@@ -0,0 +1,71 @@
+const productsContainer = document.querySelector(".products-container");
+const productDetailsContainer = document.querySelector(".product-details");
+
+async function displayProducts() {
+  const response = await fetch("http://localhost:3000/api/products");
+  const { data: products } = await response.json();
+  const firstTenProducts = products.slice(0, 10);
+  productsContainer.innerHTML = firstTenProducts
+    .map(
+      (product) => `
+    <div class="product-card" data-id="${product.id}">
+      <img class="product-image" src="${product.images}">
+      <div class="product-wrapper-name">
+        <p class="product-name">${product.title}</p>
+      </div>
+      <div class="product-wrapper-price">
+        <p class="product-price">$${product.price}</p>
+      </div>
+      <div class="product-wrapper-button">
+        <button class="product-button">Delete</button>
```

Comment:
bad name of the class ("product-button"). especially since you're accessing it from js to remove the element. 
'product-delete-button' is better.